mscore/preferenceslistwidget.cpp

```diff
@@ -112,8 +112,8 @@ std::vector<QString> PreferencesListWidget::save()
 //---------------------------------------------------------
 
 PreferenceItem::PreferenceItem()
-{
-}
+      {
+      }
```

Comment:
An empty default constructor to me is something that is either not needed at all or belongs into the cirrespoinding header file

Reply:
I agree, I will move it to the header file.